src/main/scala/com/typesafe/sbt/osgi/Osgi.scala

```diff
@@ -86,8 +86,12 @@ private object Osgi {
       val manifest = jar.getManifest
       jar.writeFolder(tmpArtifactDirectoryPath)
 
-      def content =
-        sbt.Path.contentOf(tmpArtifactDirectoryPath).filterNot { case (_, p) => p == "META-INF/MANIFEST.MF" }
+      def content = {
+        import _root_.java.nio.file.*
```

Comment:
Can we use `._` instead of `.*` here so its consistent with the rest of the project?

Otherwise lgtm